python/cuda_cccl/README.md

```diff
@@ -0,0 +1,11 @@
+# `cuda.cccl`: Experimental CUDA Core Compute Library Python module with CCCL headers
```

Comment:
Nit: we should consider the name `cuda.cccl_headers` for clarity.

Reply:
I have reservation on this, if considering the mirroring to conda packages (https://github.com/NVIDIA/cccl/pull/3201#discussion_r1893115804).